packages/mui-joy/src/Snackbar/SnackbarProps.ts

```diff
@@ -47,12 +55,12 @@ export type { SnackbarCloseReason } from '@mui/base/useSnackbar';
 
 export interface SnackbarTypeMap<P = {}, D extends React.ElementType = 'div'> {
   props: P &
-    Omit<UseSnackbarParameters, 'open'> & {
+    UseSnackbarParameters & {
```

Comment:
I had omitted `open` from `useSnackbar` because I wanted to make the `open` prop required. What do you think about making the open prop mandatory?

Reply:
To be honest, I don't have much preference for this so I'd lean on Base instead. @mnajdova What's your thought?